data_management/opensearch_indexer/tests/test_consignment_lambda_handler.py

```diff
@@ -311,6 +312,8 @@ def test_lambda_handler_invokes_bulk_index_with_correct_file_data(
                     "file_id": str(file_3_id),
                 },
             ]
+
+            unittest.TestCase().assertCountEqual(args[0], expected_list)
```

Comment:
I've never seen `assertCountEqual` before and just read up on it, very cool, it works for unhashable objects unlike the option I was going to suggest of casting to set and asserting on sets!